rollup-namespaced/sfdx-project.json

```diff
@@ -4,8 +4,8 @@
             "default": true,
             "package": "apex-rollup-namespaced",
             "path": "rollup-namespaced/source/rollup",
-            "versionName": "Spring 23 updates continued",
-            "versionNumber": "1.0.22.0",
+            "versionName": "Adding Group By capabilities to rollups",
```

Comment:
You... might actually be able to consolidate your 2 `sfdx-project.json` files into just a single file again (which might simplify your pipeline?). 

- For Nebula Logger, I have to use 2 `sfdx-project.json` files because I have some legacy/deprecated metadata that's still included in the managed package, but for your 2 packages, the metadata is 100% the same. 
- For Nebula Cache Manager, I was able to put [everything into a single file](https://github.com/jongpie/NebulaCacheManager/blob/v1.0.1/sfdx-project.json) without any issues.

Not sure if you want to try to change of this now, but just thought that I'd mention it since it could help simplify your build jobs

Reply:
I'll have to take a look! there were some issues with the namespaced package - like the test suite referring to class names without the namespace - that I would still have to address, but that certainly sounds promising!

Reply:
Oooooh, right, I forgot about test suites being problematic! Yeah you'd have to do something to handle those, so maybe it's not worth the change

Reply:
I still may come back to this, I haven't decided. One thing I _like_ about having the namespaced `sfdx-project.json` is it allows for a long tail as far as versions being kept in the file (whereas the primary one I feel like I have to keep pruned every few releases). Pros and cons to each approach - I can't decide which one I like more!